src/pci-gateway/src/main/java/ca/bc/gov/open/jag/pcigateway/api/RedirectController.java

```diff
@@ -38,26 +40,21 @@ public RedirectController(AppProperties appProperties, RestTemplate restTemplate
         this.restTemplate = restTemplate;
     }
 
-    @GetMapping("/pcigw/Payment/Payment.asp")
-    public RedirectView localRedirect(HttpServletRequest request) throws MissingServletRequestParameterException {
+    @GetMapping({"/{[P-p]ayment}/{[P-p]ayment\\.asp","/{[P-p]ayment[P-p]rofile}/{[W-w]ebform\\.asp}"})
+    public RedirectView paymentRedirect(HttpServletRequest request) throws MissingServletRequestParameterException {
 
         logger.info("received new Payment redirect request");
```

Comment:
update the text maybe to capture the redirect, might not always be a payment now

Reply:
Yeah, I will update